### DISCUSSION TITLE
電動車

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#991] [General]
近幾年電動車越來越普及，不管是特斯拉或是各大機車廠家推出的電動機車(像是gogoro之類的)，都在暗示著電動車的時代即將來臨，但是我覺得很多人在讚美電動車的過程中都忽略了它的弊端，就舉最簡單的，大家都覺得電動車比較環保不會排放二氧化碳，其實不然阿，他只是在產生電力的時候用火力發電產生電和二氧化碳而已，比起機車來說，神智還有可能總碳排放量還更多，加上電動機車很安靜，雖然近期電動機車都有配備起步會有聲音提醒，但是車禍的情況還是層出不窮，還有很多電動車的弊端無法一一舉例，記得任何事情都是有一體兩面的不要一昧地去讚美雸忽略了它的缺點
以上

**Comment by ruthchen1020** [comment]
就看火力發電廠的熱效率加傳輸時的損耗總效率，跟油車引擎的總效率，比一比。
順便考慮下排出的氣體成分。

**Comment by ruthchen1020** [comment]
另外我覺得油電混合比較期待，因為前幾天的新聞說，韓國天氣讓汽車電池蓄電量降低50%，感覺電池比引擎難做還汙染環境。

**Comment by Floriaand** [comment]
我在買機車的時候有考慮電動車，但是真的太重了，我其他買機車的同學也有抱怨這一點，我覺得這是它最大的缺點。

**Comment by WEI-CHEN-CHUN** [comment]
目前來說產生電的方式，仍是不環保的，但是每一台傳統車就是一個汙染源，而火力發電廠雖也會汙染，但仍透過某些方法來降低。

**Comment by wenchi9023** [comment]
有想買電動機車，但想到現在我生活圈中的充電站還不像加油站一樣普及就打退堂鼓了

**Comment by Jasper525** [comment]
電動車要持續發展的話，需要注重電池的開發。不然大量開採鋰電池需要的稀土元素，只會造成更多碳排。

**Comment by Doreen331** [comment]
之前買車的時候有在考慮電動車，但現在的充電樁還不普及，加上我認為技術還沒到非常成熟的地步，若考慮到環保因素，如你所說，電動車也不一定比較環保，若是排除油車的選項，我個人認為hybrid會是一個更好的選擇

**Comment by s96015welson** [comment]
我認為電動機車的電池是很大的問題，不僅發生火災時滅火不易，未正確回收造成的汙染也很嚴重，需要更慎重對待，因此短期內我不會積極去選擇電動車。

**Comment by Twenty0522** [comment]
或許電動車與油車對環境造成的污染量相同，但我們可以考慮什麼樣形式的污染我們比較可以接受。像是我很喜歡電動車在噪音方面的污染降低許多，減少了大城市的喧囂，而我們也須要培養用路人的安全觀念。

**Comment by ru0372572** [comment]
我覺得就是要慢慢轉型吧，一時絕對無法把所有東西從A變成B，尚未完善的計畫，需要時間慢慢找到經濟與環境成本最低的方法。就跟核能一樣，說要關掉燃煤，以核電取代，短時間內也是不可能一樣。

**Comment by YANRONG0123** [comment]
我覺得不論是油動車或是電動車，我們應該去尋找的是如何降低碳排並永續所使用能源的方法，哪種有利就用哪一種，重視環保的情況下同時經濟性也不可忽略。

**Comment by 20050201Shawn** [comment]
我覺得可以發展看看太陽能電動車(?)，或是其他可再生能源，不然在現今這世代電子產品過於普及，用電量大得可觀，能源消耗也是個大問題。

**Comment by happy246810** [comment]
我覺得雖然電動車在行進的過程中不會產生炭排，但在產生電的過程中也是有炭排。我認為現在的趨勢是盡量提升再生能源的比例，降低碳排放，這才是現在社會所追求的方向。

**Comment by LinChloe** [comment]
感覺電動車牽車時比較重，但它的顏色比起耗油的都好看很多，每個顏色看了都令人好想買下去，可惜我沒有錢。

**Comment by thomasxshen** [comment]
電動車隨著技術的進步和對環境保護的增加意識，越來越受到消費者的歡迎。電動車不使用化石燃料，排放更少的溫室氣體和其他污染物，有助於減少空氣污染和對氣候變化的影響，但儘管技術在不斷進步，但大多數電動車的續航里程仍然低於傳統汽車，長途旅行可能需要計劃充電站停留，相對於加油只需幾分鐘，充電可能需要幾小時，這對於需要快速補充能源的用戶來說可能是一個問題。

**Comment by William08290829** [comment]
雖然電動車的二氧化碳排放量需要考慮到製造電所產生的二氧化碳，但相信隨著綠色能源的發展，未來會找到更多環保的發電方法。這樣一來電動車也許就有可能是一個相對較好的選擇了。

**Comment by Eileen050824** [comment]
當初家裡的確有考慮要不要換電動車，畢竟這絕對是未來一大趨勢。但考量到價格高(政府補助沒有很多)和充電站實在不多，家人先暫時作罷。電動車的好處像作者說得一樣，但台灣的發展仍有更好的空間，我也相信等設施和相關規定完善後，電動車也能變成交通主流！

**Comment by yyujimmy** [comment]
你提到的觀點是很值得關注的，確實電動車在環保和創新方面有著很多優勢，但同時也存在一些挑戰和問題需要面對。

能源來源的問題： 雖然電動車本身不直接排放尾氣，但其實隨著能源生產的過程，如火力發電廠或燃煤發電廠，仍然可能排放大量的二氧化碳和其他污染物。因此，能源來源的清潔度對於電動車的整體環保效益至關重要。
資源和製造問題： 電動車的電池生產需要大量的鋰等金屬資源，這些資源的開採和製造過程也可能對環境產生負面影響。此外，電池的回收和處理也是一個挑戰，因為目前還沒有完全有效的回收技術。
安全問題： 雖然電動車通常在行駛時比較安靜，但這也可能增加行人和其他車輛的意外風險，尤其是在行人過馬路時可能不易察覺。另外，電動車的電池也存在著一定的安全風險，如過熱或充電時可能發生的故障。
總的來說，電動車確實是一種有潛力的替代交通方式，但我們需要全面考慮其影響，並努力解決其中存在的問題，以實現真正的環保和可持續發展。

**Comment by willychiuchiu** [comment]
確實電動車的普及帶來了環保和技術進步，但也必須正視其可能存在的負面影響，例如電力生產的排放和安全問題等。持續的討論和評估是必要的，以確保電動車發展的方向符合整體社會和環境的利益。

**Comment by KH1126** [comment]
電動車確實具有環保、節能等優勢，但同時也存在著一些挑戰和問題需要解決。
雖然電動車發展還未成熟，但基本確定是未來的趨勢。如果政府在設施和相關規定有更完善計畫情況下，
我相信電動車會變成交通主流！

**Comment by 110101501** [comment]
雖然電動車具備環保之效應，但要選取電動車或是油車，也要考量使用者騎乘的情形。電動車雖然美觀環保，但都會有月資費，即便沒什麼在騎乘卻要支付一定的金額。對於不常騎車的人而言，選取電動車性價比就沒有這麼高了。

**Comment by YuanWang08** [comment]
現在電動車最大的問題就是電池的處理，不同於傳統燃油汽車，若是電動車著火，是不能夠撲滅的，只能等他燒完，如果發生在地下室，就只能等全部的車一起陪葬，因為那些電池的材料是不能夠用一般方式滅火的。除此之外，還有電池壽命的問題，像是gogore第一批生產的電池差不多已經到了要退役的時間了，電池的處理更是不環保。

**Comment by yxin214** [comment]
如同版主所說，普遍認為電是乾淨的能源，但電終究是由火力產生，能量轉換過程必定會產生更多浪費。但我認為，隨著人類技術越來越進步，一定會找到最適合做為車輛發動的能源。

**Comment by YOGA012** [comment]
雖然電動車也有其弊端，但我仍然偏好電動車。首先，它們在運行過程中不會直接排放有害氣體，對於城市空氣質量有明顯的改善。其次，隨著可再生能源的普及，電動車的碳排放問題也會逐漸減少。此外，電動車的技術不斷進步，例如增加行車音效來提高安全性。電動車的安靜、節能和科技感都是我選擇它們的原因。雖然它們並非完美，但我相信隨著技術的進步，這些問題會逐漸得到解決。電動車的未來值得期待。

**Comment by alexender0104** [comment]
我認為電動車需考慮到一個點，那就是他的放棄電池該如何處理，許多人常常說著電動車較為環保，但電動車使用的鋰離子電池含有大量的貴金屬和有害物質，如果處理不當，可能對環境和人類健康造成危害。

**Comment by JustAHedgehog** [comment]
關於電動車產生電力會產生更多二氧化碳的問題，這邊建議大家可以查看[這篇文章](https://8billiontrees.com/carbon-offsets-credits/carbon-ecological-footprint-calculators/car-calculator/)，這邊他顯示電動車的產生的碳足跡，目前確實主流還是以火力的方式產生電力，但其實一旦使用其他能源去產生電力的話會明顯比原本油車產生更少的碳足跡，如下圖所示：
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/143854000/0bdaf981-9eb1-413c-81c9-f505449a8d1d)

所以去購買電動車本身並沒有錯誤，只是還是需要等到產電方式轉換再買而已，個人認為

**Comment by JustAHedgehog** [comment]
對了，針對碳排放議題除了選擇純電動車以外，這邊也推薦購買郵電混和車，原因可以由[此網站](https://elements.visualcapitalist.com/life-cycle-emissions-of-electric-hybrid-and-combustion-engine-vehicles/)得知：
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/143854000/7c7dd8a4-32a5-4af4-a650-8030e20d4e37)
這是一篇2023年6月23日的各報告，在這邊已經發現郵電混和車的碳排放已經有明顯下降，想必再過幾年淨零碳排是可行的，至於充電問題，其實現在也有推出所謂的Full Hybrids (FHRVs) 款式，特色就是利用原本的燃燒系統幫助電池充電，但油耗以及碳排放也有減低，所以個人滿推薦的。---By 昨天才報告【油電混和車與碳足跡的關係】的小白@@

**Comment by hbh10601** [comment]
目前我個人還是使用油車，畢竟看到很多電動車的電池出問題(斷電、找不到換電站)，外觀什麼的還是不夠吸引我去換成電動車

**Comment by Shang621** [comment]
電動車是利用電力驅動的交通工具，被視為未來汽車工業的趨勢。它們不僅環保，減少了對化石燃料的依賴，還降低了空氣污染和噪音污染。電動車的技術不斷進步，續航里程和充電速度逐漸提升，使其成為越來越具競爭力的替代品。政府的政策支持和消費者對環保意識的增強也推動了電動車市場的快速發展。

**Comment by sandyho0307** [comment]
電動車的普及確實是未來交通發展的一個重要趨勢，但正如你提到的，我們在讚美電動車的同時，也不能忽視它所存在的一些弊端。電動車雖然在運行過程中不排放尾氣，看似對環保有巨大貢獻，但產生所需電力的過程中仍可能涉及到污染問題，這是很多人常常忽略的一點。
